tasks/configure.yml

```diff
@@ -28,11 +28,12 @@
 - name: configure prometheus static targets
   template:
```

Comment:
Instead of a template, why not `copy` with `content: "item.value | to_nice_yaml(indent=2)"`

Reply:
This won't allow setting `{{ ansible_managed }}` comment as the first line.

Reply:
Would this work?

```yaml
content: |
  # {{ ansible_managed }}
  {{ item.value | to_nice_yaml(indent=2) }}
```

Reply:
No you can't. `ansible_managed` is available only in `template` module.
https://github.com/ansible/ansible/issues/34693
http://docs.ansible.com/ansible/latest/template_module.html#synopsis

Reply:
Bleh, ok, maybe something to fix in Ansible upstream, there are many cases where it would be nice if Ansible would simply write out a nice yaml file.

Reply:
Yeah, it could help in some situations. Go to https://github.com/ansible/ansible/issues/34693 and maybe we could change it into feature request :smile: